### PR TITLE
fix broken tests and minor Check Entitlement improvements

### PR DESCRIPTION
**Pull request by YannanGao-gs**
## Summary

- [x] fix broken tests from https://github.com/finos/legend-engine/pull/1437
- [x] minor Check Entitlement improvements

## How did you test this change?

- [ ] Test(s) added
- [x] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

### CODE REVIEW

File: .changeset/twelve-geese-tan.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-query-builder': patch
+---
+
+Minor Check Entitlement improvments
```
Comment (akphi) [inline .changeset/twelve-geese-tan.md:5]: we dont need this message here

Comment (akphi) [inline .changeset/twelve-geese-tan.md:5]:
```suggestion
```

**Review by akphi — APPROVED** [review]

File: packages/legend-query-builder/src/components/QueryBuilder.tsx [hunk 2]
```diff
@@ -506,33 +506,31 @@ export const QueryBuilder = observer(
                           </MenuContentItemLabel>
                         </MenuContentItem>
                         <MenuContentDivider />
-                        <MenuContentItem onClick={editQueryInPure}>
+                        <MenuContentItem
+                          onClick={openCheckEntitlmentsEditor}
+                          disabled={
+                            queryBuilderState.isQuerySupported &&
+                            queryBuilderState.fetchStructureState
```
Comment (MauricioUyaguari) [inline packages/legend-query-builder/src/components/QueryBuilder.tsx:513]: i thought we were changing this to allow graph fetch as well?

**Comment by hardikmaheshwari** [comment]
Thanks @YannanGao-gs for fixing this, would be interested to know if I can do something on my part to detect such failures. Thanks again

**Pull request merged**